lib/kafo/configuration.rb

```diff
@@ -135,7 +134,7 @@ def modules
     end
 
     def module(name)
-      modules.find { |m| m.name == name }
+      modules.find { |m| m.identifier == name }
```

Comment:
This looks like a bugfix. Could you provide a bit more info about why it's wrong?

Reply:
I tried to update the commit message with more information. The most telling is to look at the hook context documentation. Unfortunately, I think `name` in some areas of code refers to the Puppet naming scheme with `::` and in other places it references this internal "name" construct that Kafo creates -- https://github.com/theforeman/kafo/blob/master/lib/kafo/puppet_module.rb#L175